### ISSUE TITLE
Run-AlValidation fails when -InstallApps is used

### ISSUE THREAD

**Issue by AskeHolst** [microsoft/navcontainerhelper#1599]
Run-AlValidation fails on removing a temp folder when -InstallApps is used.
I found with my real world apps, but recreated it with a simple setup using two "Hello World" apps. This will of course fail later in the process, but just to make repro simple.

**Scripts used to create container and cause the issue**
```
Run-AlValidation `
    -licenseFile $licenseFile `
    -apps @("E:\Temp\Hello2\Default publisher_Hello2_1.0.0.0.app") `
    -affixes @("TEST") `
    -countries @("gb", "dk") `
    -installApps @("E:\Temp\Hello1\Default publisher_Hello1_1.0.0.0.app") 
```

**Full output of scripts**
BCContainerHelper is version 1.0.19
BCContainerHelper is running as administrator
Host is Microsoft Windows 10 Enterprise - 20H2
Docker Client Version is 20.10.0
Docker Server Version is 20.10.0
```
PS C:\WINDOWS\system32> E:\Temp\AppValidTest.ps1
BcContainerHelper version 1.0.19
  _____                               _                
 |  __ \                             | |               
 | |__) |_ _ _ __ __ _ _ __ ___   ___| |_ ___ _ __ ___ 
 |  ___/ _` | '__/ _` | '_ ` _ \ / _ \ __/ _ \ '__/ __|
 | |  | (_| | | | (_| | | | | | |  __/ |_  __/ |  \__ \
 |_|   \__,_|_|  \__,_|_| |_| |_|\___|\__\___|_|  |___/

Container name               bcserver
Image name                   
Credential                   admin/Zanu6693
License file                 Specified
MemoryLimit                  8G
validateVersion              
validateCurrent              False
validateNextMinor            False
validateNextMajor            False
SasToken                     Not Specified
countries                    gb,dk
validateCountries            gb,dk
affixes                      TEST
supportedCountries           
vsixFile                     
Install Apps
- E:\Temp\Hello1\Default publisher_Hello1_1.0.0.0.app
Previous Apps
- None
Apps
- E:\Temp\Hello2\Default publisher_Hello2_1.0.0.0.app
  _____       _                      _                         _   _  __           _       
 |  __ \     | |                    (_)                       | | (_)/ _|         | |      
 | |  | | ___| |_ ___ _ __ _ __ ___  _ _ __   ___    __ _ _ __| |_ _| |_ __ _  ___| |_ ___ 
 | |  | |/ _ \ __/ _ \ '__| '_ ` _ \| | '_ \ / _ \  / _` | '__| __| |  _/ _` |/ __| __/ __|
 | |__| |  __/ |_  __/ |  | | | | | | | | | |  __/ | (_| | |  | |_| | || (_| | (__| |_\__ \
 |_____/ \___|\__\___|_|  |_| |_| |_|_|_| |_|\___|  \__,_|_|   \__|_|_| \__,_|\___|\__|___/
                                                                                           
Found https://bcartifacts.azureedge.net/sandbox/17.2.19367.20615/gb
Found https://bcartifacts.azureedge.net/sandbox/17.2.19367.20615/dk
Using https://bcartifacts.azureedge.net/sandbox/17.2.19367.20615/gb
  _____       _                      _       _                   _                           _                       
 |  __ \     | |                    (_)     (_)                 | |                         | |                      
 | |  | | ___| |_ ___ _ __ _ __ ___  _ _ __  _ _ __   __ _    __| | ___ _ __   ___ _ __   __| | ___ _ __   ___ _   _ 
 | |  | |/ _ \ __/ _ \ '__| '_ ` _ \| | '_ \| | '_ \ / _` |  / _` |/ _ \ '_ \ / _ \ '_ \ / _` |/ _ \ '_ \ / __| | | |
 | |__| |  __/ |_  __/ |  | | | | | | | | | | | | | | (_| | | (_| |  __/ |_) |  __/ | | | (_| |  __/ | | | (__| |_| |
 |_____/ \___|\__\___|_|  |_| |_| |_|_|_| |_|_|_| |_|\__, |  \__,_|\___| .__/ \___|_| |_|\__,_|\___|_| |_|\___|\__, |
                                                      __/ |            | |                                      __/ |
                                                     |___/             |_|                                     |___/ 
Remove-Item : Cannot find path 'C:\Users\AskeHolst\AppData\Local\Temp\b63dd82d-bd60-4133-b0d2-18df18c8a5d2' because it does not exist.
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.19\AppHandling\Run-AlValidation.ps1:190 char:9
+         Remove-Item $tmpFolder -Recurse -Force
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\AskeHo...d2-18df18c8a5d2:String) [Remove-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand
 
...
```

**Additional context**
Sample apps used in this test are attached.
[HelloApps.zip](https://github.com/microsoft/navcontainerhelper/files/5775927/HelloApps.zip)

**Comment by freddydk** [comment]
I think this is fixed in the latest preview (1.0.20 -preview)

**Comment by freddydk** [comment]
Indeed it is - just tested this.

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.0
